@hubblecommerce/hubble/core/components/customer/TheCustomerContext.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="search-context-wrapper">
```

Comment:
correct class name to customer-context-wrapper instead of establish new class customer-flyout